pages/studentAssessmentExam/studentAssessmentExam.js

```diff
@@ -8,25 +8,43 @@ var sqldb = require('../../prairielib/lib/sql-db');
 var sqlLoader = require('../../prairielib/lib/sql-loader');
 
 var sql = sqlLoader.loadSqlEquiv(__filename);
+var groupAssessmentHelper = require('../shared/studentGroupAssessmentHelpers/studentGroupAssessmentHelpers');
 
 router.get('/', function (req, res, next) {
   if (res.locals.assessment.type !== 'Exam') return next();
+  var params = {
+    assessment_id: res.locals.assessment.id,
+    user_id: res.locals.user.user_id,
+  };
   if (res.locals.assessment.multiple_instance) {
-    res.render(__filename.replace(/\.js$/, '.ejs'), res.locals);
+    if (res.locals.assessment.group_work) {
+      groupAssessmentHelper.getConfigInfo(req, res, function () {
+        res.render(__filename.replace(/\.js$/, '.ejs'), res.locals);
+      });
```

Comment:
You'll need to handle errors correctly here:
```suggestion
      groupAssessmentHelper.getConfigInfo(req, res, function (err) {
        if (ERR(err, next)) return;
        res.render(__filename.replace(/\.js$/, '.ejs'), res.locals);
      });
```
Ditto for everywhere else.

Reply:
Error handling added in [4baf174](https://github.com/PrairieLearn/PrairieLearn/pull/5290/commits/4baf1740683c146a1b89ad57c707d5f3942bc555) and [912faf1](https://github.com/PrairieLearn/PrairieLearn/pull/5290/commits/912faf19632e1d86046f17e6c1dbbad9769454e2)